node/core/backing/src/lib.rs

```diff
@@ -436,7 +437,7 @@ impl CandidateBackingJob {
 						panic!("`self` hasn't dropped and `self` holds a reference to this sender; qed");
 					}
 				}
-				to_job = self.rx_to.next() => match to_job {
+				to_job = rx_to.next() => match to_job {
```

Comment:
I don't get the difference to what there was before? So in what way does it change the execution?

Reply:
It improves the ownership story so nowhere else can poll `rx_to`, leading us to poll it after completion.